pkg/flux/logs_test.go

```diff
@@ -110,48 +103,26 @@ func TestSetup(t *testing.T) {
 }
 
 func TestSetupFluxBin(t *testing.T) {
-	version.FluxVersion = "0.11.0"
-	SetupFluxBin()
-	homeDir, err := shims.UserHomeDir()
+	dir, err := ioutil.TempDir(t.TempDir(), "a-home-dir")
 	require.NoError(t, err)
-
-	fluxPath := fmt.Sprintf("%v/.wego/bin", homeDir)
-	require.DirExists(t, fluxPath)
-	binPath := fmt.Sprintf("%v/flux-%v", fluxPath, version.FluxVersion)
-	require.FileExists(t, binPath)
-
-	version.FluxVersion = "0.12.0"
-	SetupFluxBin()
-	require.NoFileExists(t, binPath)
-	binPath = fmt.Sprintf("%v/flux-%v", fluxPath, version.FluxVersion)
-	require.FileExists(t, binPath)
+	defer os.RemoveAll(dir)
+	override.WithOverrides(func() override.Result {
```

Comment:
I added this because I noticed the test was running against the user's actual home dir. Possibly we should pull out all "os" operations (UserHomeDir, Exit, Stat, etc.) into a separate client object for mocking :thinking: